lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -1668,3 +1668,25 @@ def norm(x):
         colors = np.asanyarray(color).copy()
 
     return colors
+
+
+class Arrow3D(FancyArrowPatch):
+    """
+    3D FancyArrowPatch object.
+    """
+    def __init__(self, xs, ys, zs, *args, **kwargs):
+        """
+        Parameters
+        ----------
+            xs, ys, zs : array-like
+                The x, y, and z coordinates of the arrow's start and end points.
+        """
+        super().__init__((0,0), (0,0), *args, **kwargs)
+        self._verts3d = xs, ys, zs
+
+    def do_3d_projection(self, renderer=None):
+        """Projects the points according to the renderer matrix."""
+        xs3d, ys3d, zs3d = self._verts3d
+        xs, ys, zs = proj3d.proj_transform(xs3d, ys3d, zs3d, self.axes.M)
+        self.set_positions((xs[0],ys[0]),(xs[1],ys[1]))
```

Comment:
Looks like I missed this before, but there has to be spaces following the commas here.

```suggestion
        self.set_positions((xs[0], ys[0]), (xs[1], ys[1]))
```